### PR TITLE
Adding api

### PR DESCRIPTION
**Pull request by hswiergiel**

### CODE REVIEW

**Review by Sax593 — APPROVED** [review]

File: frontend/src/pages/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.js
```
Comment (LoicBrassart) [inline frontend/src/pages/.gitignore:1]: ?

File: frontend/package.json [hunk 2]
```diff
@@ -10,9 +10,11 @@
     "fix": "prettier --write . && eslint --fix --ext .jsx ."
   },
   "dependencies": {
+    "axios": "^1.1.3",
     "prop-types": "^15.8.1",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-player": "^2.11.0",
```
Comment (LoicBrassart) [inline frontend/package.json]: I didn't find it's use on your project. Should it be deleted ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evils to fix

File: frontend/src/pages/.gitignore (hunk 1 above)
Comment (hswiergiel) [reply frontend/src/pages/.gitignore:1]: c'est un fichier test que je laisse sur mon pc pour garder du code qui me servira plus tard, vu qu'on ne peut pas envoyer de code mort. is it ok?

File: frontend/package.json (hunk 2 above)
Comment (hswiergiel) [reply frontend/package.json]: yes, it was just a try to use it. May we keep it as Valentin know he's going to need it after?

File: frontend/src/pages/.gitignore (hunk 1 above)
Comment (LoicBrassart) [reply frontend/src/pages/.gitignore:1]: Yeah ok, fine by me!

File: frontend/package.json (hunk 2 above)
Comment (LoicBrassart) [reply frontend/package.json]: I'd prefer you to delete it and Valentin can install it as needed

Comment (hswiergiel) [reply frontend/package.json]: ok done! i've just merged it with the branch which has just been added to dev

**Review by Sax593 — APPROVED** [review]

**Comment by hswiergiel** [comment]
regex commit has been merged before this one and already contains the connection to API

**Pull request closed without merging**